docs/stability.md

```diff
@@ -0,0 +1,38 @@
+---
+title: API Stability
+sort_rank: 8
+---
+
+# API Stability Guarantees
+
+Prometheus promises API stability within a major version, and strives to avoid
+breaking changes for key features. Some features which are cosmetic, still
```

Comment:
Comma before "which".

Reply:
"which" starts a non-restrictive relative clause, which (according to most systems of English punctuation) need to be enclosed in commas. This, BTW, means we also need a comma after "services", which I missed to mention. Perhaps that clarifies things.

On the other hand, we don't need to invest time into this. If it is not clear to you at this point, either trust me or leave it. No strong feelings from my side about it, it's just a nit.

Reply:
I see what you mean now, changed.